site/pages/create-social-media-post/[disruptionId]/[socialMediaPostIndex].page.tsx

```diff
@@ -75,7 +75,10 @@ const CreateSocialMediaPost = (props: CreateSocialMediaPostPageProps): ReactElem
                                 stateUpdater(props.disruptionSummary, "messageContent");
                             }}
                         >
-                            <p className="text-govBlue govuk-body-m">Copy from disruption summary</p>
+                            {!pageState.inputs.messageContent ||
+                            (pageState.inputs && pageState.inputs.messageContent.length === 0) ? (
+                                <p className="text-govBlue govuk-body-m">Copy from disruption summary</p>
```

Comment:
If the message content has an error and then this button is clicked - the summary is displayed but the error is not removed

Reply:
cannot be fixed due to onblur error

Reply:
done

Reply:
adjusted based on discussion